src/operator/controllers/iam/serviceaccounts/serviceaccount_controller_test.go

```diff
@@ -252,36 +251,37 @@ func (s *TestServiceAccountSuite) TestServiceAccountSuite_CreateIAMRole_NoArn_Pa
 	s.Require().NoError(err)
 }
 
-func (s *TestServiceAccountSuite) TestServiceAccountSuite_CreateIAMRoleWithMarkAsUnusedLabel_SameArn() {
-	req := ctrl.Request{
-		NamespacedName: types.NamespacedName{Namespace: testNamespace, Name: testServiceAccountName},
-	}
-
-	serviceAccount := corev1.ServiceAccount{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:        testServiceAccountName,
-			Namespace:   testNamespace,
-			Annotations: map[string]string{metadata.ServiceAccountAWSRoleARNAnnotation: testRoleARN},
-			Labels: map[string]string{
-				metadata.OtterizeServiceAccountLabel: metadata.OtterizeServiceAccountHasPodsValue,
-				mockServiceManagedByLabel:            "true",
-				awsagent.OtterizeAWSUseSoftDeleteKey: "true",
-			},
-			Finalizers: []string{metadata.IAMRoleFinalizer},
-		},
-	}
-	s.client.EXPECT().Get(gomock.Any(), req.NamespacedName, gomock.AssignableToTypeOf(&serviceAccount)).DoAndReturn(
-		func(arg0 context.Context, arg1 types.NamespacedName, arg2 *corev1.ServiceAccount, arg3 ...client.GetOption) error {
-			serviceAccount.DeepCopyInto(arg2)
-			return nil
-		},
-	)
-
-	s.mockIAM.EXPECT().OnServiceAccountUpdate(gomock.Any(), gomock.AssignableToTypeOf(&serviceAccount)).Return(false, false, nil)
-
-	_, err := s.reconciler.Reconcile(context.Background(), req)
-	s.Require().NoError(err)
-}
+//
```

Comment:
Normally I would delete this, but left this here to make it easier to revive it in a few days